content/docs/1.1.0/high-availability/recover-volume.md

```diff
@@ -3,99 +3,9 @@
-
-
-## Requirements
-
-The auto remount does not work for `xfs` filesystem.
-
-Mounting one more layers with the `xfs` filesystem is not allowed and will trigger the error `XFS (sdb): Filesystem has duplicate UUID <filesystem UUID> - can't mount`.
-
-If you use the `xfs` filesystem, you will need to manually unmount, then mount the `xfs` filesystem on the host. The device path on the host for the attached volume is `/dev/longhorn/<volume name>` . 
-
-## Automatically Remount Volumes and Restart Workloads
-
-In order to recover unexpectedly detached volumes automatically, set `restartPolicy` to `Always`, then add `livenessProbe` for the workloads using Longhorn volumes.
-
-Then those workloads will be restarted automatically after reattachment and remount.
-
-Here is one example for the setup:
-
-```
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: longhorn-volv-pvc
-spec:
-  accessModes:
-    - ReadWriteOnce
-  storageClassName: longhorn
-  resources:
-    requests:
-      storage: 2Gi
----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: volume-test
-  namespace: default
-spec:
-  restartPolicy: Always
-  containers:
-  - name: volume-test
-    image: nginx:stable-alpine
-    imagePullPolicy: IfNotPresent
-    livenessProbe:
-      exec:
-        command:
-        - ls
-        - /data/lost+found
-      initialDelaySeconds: 5
-      periodSeconds: 5
-    volumeMounts:
-    - name: volv
-      mountPath: /data
-    ports:
-    - containerPort: 80
-  volumes:
-  - name: volv
-    persistentVolumeClaim:
-      claimName: longhorn-volv-pvc
-```
-- The directory used in the `livenessProbe` will be `<volumeMount.mountPath>/lost+found`
-- Don't set a short interval for `livenessProbe.periodSeconds`, e.g., 1s. The liveness command is CPU consuming.
-
-## Manually Restart Workload Containers
-
-This solution is applied only if:
-
-- The Longhorn volume is reattached and remounted automatically.
-- The above setup is not included when the related workload is launched. In this case, the [volume mount propagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) is not `Bidirectional`, and the Longhorn remount operation won't be propagated to the workload containers if the containers are not restarted.
-
-To restart the workload containers,
-
-1. Figure out on which node the related workload's containers are running
-```
-kubectl -n <namespace of your workload> get pods <workload's pod name> -o wide
-```
-2. Connect to the node. e.g., `ssh`
-3. Figure out the containers belonging to the workload
-```
-docker ps
-```
-By checking the columns `COMMAND` and `NAMES` of the output, you can find the corresponding container
-
-4. Restart the container
-```
-docker restart <the container ID of the workload>
-```
\ No newline at end of file
+For the orphan pods (the ones with no controller), Longhorn doesn't kill them because if Longhorn does, no one will restart them.
```

Comment:
Don't call them `orphan pods` unless it's a Kubernetes word.

Reply:
Does it count? :) 
https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#additional-note-on-deployments

Reply:
I changed it to `the pods that don't have a controller`